server/container/util/src/test/java/org/apache/james/util/retry/naming/NamingExceptionRetryHandlerTest.java

```diff
@@ -40,84 +40,81 @@
     private ExceptionRetryingProxy proxy = null;
     private RetrySchedule schedule = null;
 
-    /**
-     * @see junit.framework.TestCase#setUp()
-     */
     @Before
```

Comment:
missing @override?

Reply:
Overrides on setUp()? It's not overriden...